tasks/test/createBundles.js

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
```

Comment:
This file uses the files from a basic extension layout to create a zip of an extension. Those files are checked into `git`.

Also checked into git are files at `tasks/test/circular-extension-files` which contain:

```
extension.js
testSharedModuleCircular.js (imports circularDependency.js)
circularDependency.js (imports testSharedModuleCircular.js)
```

Every file from `example-extension` is copied into an ingored folder at `tasks/test/circular-extension`, then the `circular-extension-files` for circular dependency are pushed into the sharedModules directory of `circular-extension`, and the `extension.json` from the `circular-extension-files` stomps over `task/test/circular-extension/extension.json`

an archive is then created for both `example-extension` and `circular-extension` inside another ignored directory called `./test-dist`.

If the archiving process fails, the CI action on Github will fail.

It then copies in